### PR TITLE
Schema diagram PR

### PR DESCRIPTION
**Pull request by kkasaija**
### Milestone reached.
- Created a file named schema_based_on_diagram.sql 
- Joined tables from many-to-many relationships

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+CREATE TABLE invoices(
+  id serial primary key,
+  total_amount decimal,
+  generated_at timestamp,
+  payed_at timestamp,
+  medical_history_id int
+);
+
+CREATE TABLE invoice_items(
```
Comment (muneebulrehman) [inline schema_based_on_diagram.sql:30]: Table `invoice_items` as per the diagram provided also has a `total_price` column. Kindly add the same and then submit for a re-review.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  treatment_id int
+);
+
+CREATE TABLE medical_history_treatment(
+  medical_history_id int,
+  treatment_id int,
+  CONSTRAINT fk_medical_history FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id);
+  CONSTRAINT fk_treatments FOREIGN KEY(treatment_id) REFERENCES treatments(id);
+);
+
```
Comment (muneebulrehman) [inline schema_based_on_diagram.sql:45]: You also need to create a foreign key index on the `invoices` table reference to the `medical_history_id`. Kindly do so and then submit for a re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly use [this updated README template](https://github.com/microverseinc/readme-template) for your README file. Remember to customize the README file as per your project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (kkasaija) [reply schema_based_on_diagram.sql:45]: Thank you @muneebulrehman. 
The missing column and foreign key have been added

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth 😊,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ The Pull Request has a proper title and description
✔️ Correct git-flow

 ## Required Changes ♻️
- [ ] As stated by the previous reviewer, kindly use [an updated README template](https://github.com/microverseinc/readme-template) and make sure to contain all main headers. Remember to customize the README file as per your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,16 +1,139 @@
...
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
```
Comment (OLIPLICHE) [inline README.md]: # Optional:
- Hello, A ReadMe template is the first contact between a recruiter and your code. The recruiter will not waste time on this template if you don't provide the means to set up and run your project.
- I suggest adding the Microverse icon and the Table of content.

**Review by OLIPLICHE — APPROVED** [review]
Hi @Kasaija-Kenneth, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Commit messages are meaningful 👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (kkasaija) [reply README.md]: Thanks for the guidance. This has been implemented already
@OLIPLICHE

**Pull request merged**